js/ui/popup/controllers/setup.js

```diff
@@ -93,11 +93,11 @@
                     });
                 },
                 vault: function (callback) {
-                    try {
-                        PAPI.decryptString($scope.settings.default_vault.challenge_password, $scope.settings.vault_password);
+                    var decryptedString = PAPI.decryptString($scope.settings.default_vault.challenge_password, $scope.settings.vault_password);
+
+                    if (decryptedString && decryptedString !== false) {
```

Comment:
If the first assumption of the condition is false, the condition fails either

So a simple `if (PAPI.decryptString($scope.settings.default_vault.challenge_password, $scope.settings.vault_password) !== false)` would clarify the condition.